keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/QDistribution.java

```diff
@@ -0,0 +1,7 @@
+package io.improbable.keanu.algorithms.variational;
+
+import io.improbable.keanu.network.NetworkState;
+
+public interface QDistribution {
```

Comment:
There are no instantiations of this interface in the code base, except for your TestQDistribution which only returns a constant. Can we add some more real-world examples, and tests to accompany them?